pipelines.py

```diff
@@ -26,29 +26,28 @@ def unet(config, train_mode):
                 save_output=save_output, load_saved_output=load_saved_output)
 
     mask_postprocessed = mask_postprocessing(loader, unet, config, save_output=save_output)
-    if config.postprocessor["dilate_selem_size"] > 0:
-        mask_postprocessed = Step(name='mask_dilation',
-                                  transformer=MaskDilatorStream(
-                                      **config.postprocessor) if config.execution.stream_mode else MaskDilator(
-                                      **config.postprocessor),
-                                  input_steps=[mask_postprocessed],
-                                  adapter={'images': ([(mask_postprocessed.name, 'categorized_images')]),
-                                           },
-                                  cache_dirpath=config.env.cache_dirpath,
-                                  save_output=save_output,
-                                  load_saved_output=False)
-    detached = multiclass_object_labeler(mask_postprocessed, config, save_output=save_output)
+
     output = Step(name='output',
                   transformer=Dummy(),
-                  input_steps=[detached],
-                  adapter={'y_pred': ([(detached.name, 'labeled_images')]),
+                  input_steps=[mask_postprocessed],
+                  adapter={'y_pred': ([(mask_postprocessed.name, 'images')]),
+                           'y_scores': ([(mask_postprocessed.name, 'scores')])
                            },
                   cache_dirpath=config.env.cache_dirpath,
                   save_output=save_output,
                   load_saved_output=False)
     return output
 
 
+def unet_weighted(config, train_mode):
```

Comment:
@apyskir I really like that you used the .get_step method here. Shows how reusable steps can be.

Reply:
@jakubczakon Yes, I was so happy when I figured out how easily I can change 2 steps in `unet` pipeline. This case (or similiar) could be part of a notebook showing how to use steps.